### ISSUE TITLE
How can I train the d0 on coco_human_data  ?  (only for person detection) 

### ISSUE THREAD

**Issue by xscjun** [zylo117/Yet-Another-EfficientDet-Pytorch#218]
Thanks for your share of the great work.
    I evaluate the d0 on the coco_human_val ,and get map : 0.432.  When I trained the data with the pre-trained model (efficientdet-d0.pth)   , after 100 epoch ,only get the map 0.342,and the loss is no longer drop .
		
	I trained with the script first: 
	python train.py -c 0 --batch_size 512 --lr 1e-2 --num_workers 24 
	--num_epochs 500  --load_weights efficientdet-d0.pth

	From epoch34,the learning rate drop to 1e-3 automatically,   
	From epoch 59 ，the loss is no longer drop ,then  I changed the lr to 1e-4，  but the loss is still  no longger drop（now 100epoch but still no dropping） 。
		
How can I change the Hyperparameter， to  get a better map  ？
